code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -153,3 +153,10 @@
 	icon_state = "bhole3"
 	layer = ABOVE_LYING_MOB_LAYER
 	duration = WRAITH_BANISH_BASE_DURATION+1 //So we don't delete our contents early
+
+/obj/effect/temp_visual/order/temporary_supply_beacon
+	name = "temp supply beacon"
+	icon = 'icons/misc/mark.dmi'
+	icon_state_on = "x3"
```

Comment:
isnt this the AI debug node icon

Reply:
no idea what it is used on, i've never seen it in game and it fits pretty well